doc/parameters.md

```diff
@@ -312,14 +312,34 @@ Path to a file with X.509 certificate in the PEM format for HTTPS.
 
 Path to a file with the X.509 certificate secret key in the PEM format.
 
+### `all_proxy`, `ALL_PROXY`
+
+**Default:** no value
+**Value:** string  
+**Example:** `http://forward-proxy:80`
+
+Defines a HTTP/HTTPS proxy to be used for connecting to services if a protocol-specific proxy is not specified. 
+
 ### `http_proxy`, `HTTP_PROXY`
 
 **Default:** no value
+**Value:** string  
+**Example:** `http://forward-proxy:80`
 
 Defines a HTTP proxy to be used for connecting to HTTP services.
 
 ### `https_proxy`, `HTTPS_PROXY`
 
 **Default:** no value
+**Value:** string  
+**Example:** `https://forward-proxy:443`
```

Comment:
Nope. You still need to point this to a HTTP proxy. HTTPS is not supported.

Reply:
Ah, OK, I misunderstood, sorry! Will fix.